charts/vela-core/templates/_helpers.tpl

```diff
@@ -51,9 +51,9 @@ app.kubernetes.io/name: {{ include "kubevela.name" . }}
 app.kubernetes.io/instance: {{ .Release.Name }}
 {{- end -}}
 
-{{- define "kubevela-apiserver.selectorLabels" -}}
-app.kubernetes.io/name: {{ include "kubevela.name" . }}-apiserver
-app.kubernetes.io/instance: {{ .Release.Name }}-apiserver
+{{- define "kubevela-ux.selectorLabels" -}}
+app.kubernetes.io/name: {{ include "kubevela.name" . }}-ux
+app.kubernetes.io/instance: {{ .Release.Name }}-ux
```

Comment:
Do you mean the apiservier is ux only?

Reply:
It's different with what we talked before

Reply:
The 'ux' includes both the APIServer and UIServer.
I don't want to bother with naming here. If you have suggestion I will take it.